content-src/experiments/universal-search.yaml

```diff
@@ -187,19 +187,20 @@ graduation_report: >
   <figcaption>Participant feedback survey results.</figcaption>
   </figure>
   <p><strong>Here’s what happens next</strong></p>
-  <p>What’s next for Universal Search? Now that we’ve announced retirement plans 
-  for this add-on, we will no longer maintain it.  If you have Universal Search enabled, 
+  <p>What’s next for Universal Search? Now that we’ve announced retirement plans
+  for this add-on, we will no longer maintain it.  If you have Universal Search enabled,
   you don’t need to do anything. We’ll automatically uninstall it in the coming weeks.</p>
-  <p>The experiment we built was not a complete product experience, but it was 
-  enough to meet our learning goals. If we tried a similar experiment again, we’d 
-  pursue better result relevancy and more diverse result types. We would also likely 
-  test alternate UX treatments to see if we could affect interaction with results, or 
-  improve user acceptance of the combined Awesome Bar for those users who preferred 
+  <p>The experiment we built was not a complete product experience, but it was
+  enough to meet our learning goals. If we tried a similar experiment again, we’d
+  pursue better result relevancy and more diverse result types. We would also likely
+  test alternate UX treatments to see if we could affect interaction with results, or
+  improve user acceptance of the combined Awesome Bar for those users who preferred
   separate URL and search bars. These are questions we may examine in future experiments.</p>
-  <p>Thank you to all the Test Pilots who installed Universal Search, used it, and told 
+  <p>Thank you to all the Test Pilots who installed Universal Search, used it, and told
   us what they thought!  We are grateful for your participation.</p>
   <p>Want to try a new experiment? Visit <a href="https://testpilot.firefox.com">https://testpilot.firefox.com</a>.</p>
   <div class="float-right"><strong>-Javaun Moradi, Product Manager</strong></div>
+  <p><a href="https://medium.com/mozilla-tech/universal-search-retrospective-b39e1be5d128">Additional Universal Search analysis from Chuck Harmston.</a></p>
```

Comment:
Should we open in a new tab?

Line looks funky, maybe include it in the same div as javaun's name in a new line?

![chuck](https://d3vv6lp55qjaqc.cloudfront.net/items/2O0W2u100f0A260x0c0X/Image%202016-11-29%20at%208.41.47%20AM.png?X-CloudApp-Visitor-Id=1739947&v=f4bd8356)